src/audio/audio.cpp

```diff
@@ -251,6 +259,8 @@ bool Audio::initInput(QString inDevDescr)
         return false;
     }
 
+    privateSetGain(Settings::getInstance().getInVolume() * 0.01);
```

Comment:
Of course multiplier no longer applies here -> will be removed. Thanks @initramfs.